source/code_highlighter.html.md

```diff
@@ -0,0 +1,42 @@
+# Code Highlighter
+
+Franklin uses [Rouge](http://rouge.jneen.net/) library for code highlighting.
+
+Below You can find some examples of code highlight.
+
+## Pre content
+
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+
+## Shell
+
+~~~ shell
```

Comment:
I have a preference for Github Flavored Markdown (which uses backticks instead of tildas, among other differences).

If it's cool with you, I'd like to change the Kramdown settings to accomodate that, and update these code blocks.